src/platform/mod.rs

```diff
@@ -34,12 +40,15 @@ pub trait Platform {
     fn get_max_valid_address() -> usize;
 
     const HAS_S_MODE: bool = config::VCPU_S_MODE;
+    const NB_HARTS: usize = usize::MAX;
```

Comment:
Do you have to provide a default here? Would that work or does the compiler complain?

```suggestion
    const NB_HARTS: usize;
```

Reply:
I can remove it, but I need to define it at compile time in VirtPlatform (usize::MAX ?)